.github/scripts/register-downstream-clusters.sh

```diff
@@ -41,14 +35,17 @@ userPrincipal:
   provider: local
 EOF
 
+# Log into the 4th project listed by `rancher login`, which should be the local cluster's default project.
 echo -e "4\n" | rancher login "https://$public_hostname" --token "$token" --skip-verify
 
 rancher clusters create second --import
 until rancher cluster ls --format json | jq -r 'select(.Name=="second") | .ID' | grep -Eq "c-[a-z0-9]" ; do sleep 1; done
 id=$( rancher cluster ls --format json | jq -r 'select(.Name=="second") | .ID' )
 
 kubectl config use-context "$cluster_downstream"
-rancher cluster import "$id"
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $user
```

Comment:
Interesting, I wonder if it would help to pick an admin user:

https://github.com/manno/fleet-dev-tools/commit/f53ec4c96d9217a036d05bbb4b3a2cef4fdda31b#diff-713765b9c3a7b1fdfcfe622eba2300d80eeff5d04089de16342b77072e83efa3R23